packages/core/src/App/Components/Elements/live-chat.jsx

```diff
@@ -16,39 +16,46 @@ const LiveChat = ({ is_mobile_drawer, has_cookie_account }) => {
     }, []);
 
     React.useEffect(() => {
-        let session_variables = { loginid: '', landing_company_shortcode: '', currency: '', residence: '' };
-
-        if (has_cookie_account) {
-            const domain = window.location.hostname.includes('deriv.com') ? 'deriv.com' : 'binary.sx';
-            const client_information = Cookies.get('client_information', {
-                domain,
-            });
-            if (client_information) {
-                const {
-                    loginid,
-                    email,
-                    landing_company_shortcode,
-                    currency,
-                    residence,
-                    first_name,
-                    last_name,
-                } = JSON.parse(client_information);
-                session_variables = {
-                    ...(loginid && { loginid }),
-                    ...(landing_company_shortcode && { landing_company_shortcode }),
-                    ...(currency && { currency }),
-                    ...(residence && { residence }),
+        if (window.LiveChatWidget) {
+            window.LiveChatWidget.on('ready', () => {
+                let session_variables = {
+                    loginid: '',
+                    landing_company_shortcode: '',
+                    currency: '',
+                    residence: '',
+                    email: '',
                 };
 
-                window.LiveChatWidget.call('set_session_variables', session_variables);
+                if (has_cookie_account) {
+                    const domain = window.location.hostname.includes('deriv.com') ? 'deriv.com' : 'binary.sx';
```

Comment:
We're using `deriv.com` literal in many places now. Could we refactor it to a constant value in shared? Same for `binary.sx`.